react/features/device-selection/components/AudioOutputPreview.js

```diff
@@ -113,6 +113,7 @@ class AudioOutputPreview extends Component<Props> {
      */
     _setAudioSink() {
         this._audioElement
+            && this.props.deviceId
```

Comment:
Should we disable play if there is no device id? I'm not sure what the behavior of <Audio /> is if we don't set a sink id.

Reply:
It fallbacks to default device I think, this is what I see when playing after not setting it. There is no error or anything.